hs_core/page_processors.py

```diff
@@ -47,6 +47,10 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                 - split into two functions: get_readonly_page_context(...) and
                 get_editable_page_context(...)
     """
+    editing = request.GET.get('editing')
```

Comment:
I can't find another instance of this.  Did you accidentally commit this and was using it for debugging?

Reply:
This is being discussed in a different comment, resolving so we can focus conversation there.